src/sections/stats/sections/LRNA/components/Distribution/ChartLabel.tsx

```diff
@@ -17,7 +17,7 @@ export const ChartLabel = () => {
       animate={{ opacity: 1, y: 0 }}
     >
       <div sx={{ flex: "column", align: "center", gap: 6 }}>
-        {getAssetLogo("LRNA")}
+        {<AssetLogo id="1" />}
```

Comment:
Would it be possible to use some kind of map or a constant? So we don't have to use id as a plain "1" string